integration-tests/playbook_verifier/test_verifier.py

```diff
@@ -20,13 +30,21 @@
     ],
 )
 def test_official_playbook(filename: str):
```

Comment:
Missing pytest mark here. You added pytest mark in the first commit, but in the second commit removed it.